docs/application/native/tutorials/details/app-filtering.md

```diff
@@ -190,6 +190,7 @@ package:
 | `http://tizen.org/feature/multi_point_touch.pinch_zoom` | Specify this key, if the application requires a pinch-zoom gesture feature. | 2.2.1 |
 | `http://tizen.org/feature/multi_point_touch.point_count` | Specify this key with a specific number (`int` type), if the application requires the minimum of specified number of simultaneous touches in a multi-point touch. | 2.2.1 |
 | `http://tizen.org/feature/multimedia.media_codec`     | Specify this key, if the application requires the media codec feature. | 4.0   |
+| `http://tizen.org/feature/multimedia.stream_recorder` | Specify this key, if the application requires the stream recorder feature. | 5.5   |
```

Comment:
According to https://code.sec.samsung.net/jira/browse/ACR-1506, this feature is for both mobile and wearable.

In this page, features are listed in each table for mobile & wearable.
Would you please check again if we don't need to add this feature to the table for wearable also,
as it is added to mobile only?

Also, how about following the alphabetical order for the features?

Reply:
I will add it on wearable section Thank you:)